index.html

```diff
@@ -59,8 +58,145 @@ <h1>I'm a Software Developer</h1>
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
+    </section>
+    <section class="ex1">
+      <div>
+        <h2 class="header-2">Profesional Art Printing Data</h2>
+        <p class="para-2">A daily selection of privately
+          personalized reads; no accounts or sign-ups required. Has
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
+    </section>
+    <section class="ex1">
+      <div>
+        <h2 class="header-2">Profesional Art Printing Data</h2>
+        <p class="para-2">A daily selection of privately
+          personalized reads; no accounts or sign-ups required. Has
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
+    </section>
+  </section>
+  <!-- About myself -->
+  <section class="about_myself">
+    <h1 id="abt_mslf">About Me</h1>
+    <p id="abt">Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <button id="btn_2">Get My resume</button>
+  </section>
+  <hr>
+  <section class="skills">
+    <section class="headr">
+      <section class="face">
+        <div class="tilted_square"></div>
+        <div id="text">Languages</div>
+      </section>
+      <ul class="abt_list">
+        <li class="j">JavaScript</li>
+        <li class="j">Ruby</li>
+        <li class="j">HTML</li>
+        <li class="j">CSS</li>
+      </ul>
+    </section>
+    <section class="headr">
+      <ul class="abt_list">
+        <section class="titles-skill">
+          <div class="boxp"></div>
+          <div id="text">Frameworks</div>
+        </section>
+        <li class="j">React.js</li>
+        <li class="j">Ruby on Rails</li>
+        <li class="j">RSpec</li>
+        <li class="j">Capybara</li>
+      </ul>
+    </section>
+    <section class="headr">
 
+      <ul class="abt_list">
+        <section class="titles-skill">
+          <div class="dot"></div>
+          <div id="text">Skills</div>
+        </section>
+        <li class="j">Database Management</li>
+        <li class="j">Version Control</li>
+        <li class="j">CLI</li>
+        <li class="j">API Design</li>
+        <li class="j">Web Development</li>
+      </ul>
```

Comment:
- You have done well so far, but according to the project requirements, the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") is also a list `ul` and each of them a `li`. Please fix this 🙏